MediaBrowser.Api/Syncplay/TimeSyncService.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using MediaBrowser.Controller.Configuration;
+using MediaBrowser.Controller.Net;
+using MediaBrowser.Controller.Session;
+using MediaBrowser.Model.Services;
+using MediaBrowser.Model.Syncplay;
+using Microsoft.Extensions.Logging;
+
+namespace MediaBrowser.Api.Syncplay
+{
+    [Route("/GetUtcTime", "GET", Summary = "Get UtcTime")]
+    [Authenticated]
```

Comment:
why auth is required?

Reply:
No reason in particular. I just thought that having a session might come handy in the future.
Do you think that it should be freely available instead?

Reply:
I think we should put as few endpoints in front of authentication as possible. We currently have a few and I still think that's too many. :laughing: